apps/ledger-live-mobile/e2e/models/wallet/nftGalleryPage.ts

```diff
@@ -0,0 +1,6 @@
+import { getElementById } from "../../helpers";
+
+export default class NftGalleryPage {
+  getNftList = () => getElementById(`wallet_nft_gallery_list`);
+  getNftListItems = () => getElementById(`wallet_nft_gallery_list_item`);
```

Comment:
The name is a bit unclear. Something like `getFirstNftItem`, since I'm presuming it gets just one item?

Reply:
It does get all items.

Reply:
Done

Reply:
Ah so there are multiple items with that id? Or it just grabs a container that has all of them?